cloudant-client/src/main/java/com/cloudant/client/api/DesignDocumentManager.java

```diff
@@ -88,18 +88,20 @@
 
     /**
      * Synchronizes a design document to the Database.
-     * <p>This method will first try to find a document in the database with the same id
+     * <p>This method will first try to find a document in the database with the same {@code _id}
      * as the given document, if it is not found then the given document will be saved to the
-     * database.
-     * </p>
+     * database.</p>
      * <p>If the document was found in the database, it will be compared with the given document
-     * using {@code equals()}. If both documents are not equal, then the given document will be
-     * saved to the database and updates the existing document.
-     * </p>
-     * <p>
-     * If the design document's ID is not prefixed with _design, then the _design prefix will be
-     * added.
-     * </p>
+     * using {@link com.cloudant.client.api.model.DesignDocument#equals(Object)}. Both the

```

Comment:
We should probably provide some documentation for `DesignDocument#equals` even though it only does what you'd expect it to do.

You could use something like this, shamelessly stolen and adapted from `java.lang.String`:

"Compares this design document to the specified object. The result is true if and only if the argument is not null and is a DesignDocument with the same _id, _rev, and content"